spectral_cube/cube_utils.py

```diff
@@ -750,3 +750,80 @@ def bunit_converters(obj, unit, equivalencies=(), freq=None):
     else:
         # Slice along first axis to return a 1D array.
         return factors[0]
+
+def reproject_together(cube1, cube2):
+    '''
+    Two spectral cubes to reproject to the same wcs. 
+
+    Parameters
+    ----------
+    cube1 : SpectralCube
+        A spectral cube.
+    cube2 : SpectralCube
+        A spectral cube.
+
+    Returns
+    -------
+    cube_repr1 : SpectralCube
+        A spectral cube reprojected to the optimal wcs.
+    cube_repr1 : SpectralCube
+        A spectral cube reprojected to the optimal wcs.
+    '''
+    
+    from reproject.mosaicking import find_optimal_celestial_wcs
+    # Get wcs and shape of both cubes
+    w1 = cube1.wcs.celestial
+    s1 = cube1.shape[1:]
+    w2 = cube1.wcs.celestial
+    s2 = cube1.shape[1:]
+    # Get the optimal wcs and shape for both fields together
+    wcs_opt, shape_opt = find_optimal_celestial_wcs([(w1, s1),(w2, s2)], auto_rotate=False)
```

Comment:
minor code style fix
```suggestion
    wcs_opt, shape_opt = find_optimal_celestial_wcs([(w1, s1), (w2, s2)], auto_rotate=False)
```